riseagain47/EPIC-Mail
# Ft-user-save-draft-#164288718

#### What does this PR do?
User should be able to save an email as draft and send it later

#### Description of Task to be completed?
Have the following:
A save-as-draft button in the send message view/page
A link/button to click and view to see the list saved email drafts

#### How should this be manually tested?
GIVEN A user 
WHEN User logs into their account dashboard
THEN User should see a send message link/button which links to a simple and descriptive form with text input fields, a corresponding labels and a send message button, user should be able to also see a save-as-draft-button in order to save and send later

#### Any background context you want to provide?
User should also click on the draft link/button to see a list of saved messages

#### What are the relevant pivotal tracker stories?
Feature #164288718 - User should be able to save an email as draft and send it later

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/47803641/53668637-c2961780-3c74-11e9-9dad-e14d31e730b4.png)
